### ISSUE TITLE
TypeError: crawl() takes 2 positional arguments but 3 were given

### ISSUE THREAD

**Issue by michaelbrunnbauer** [scrapy/scrapy#6004]
After upgrading from scrapy-2.9.0 to scrapy-2.10.0, I get this exception:

```
Error caught on signal handler: <bound method CrawlManager.spider_idle of <scrapyrt.core.CrawlManager object at 0x7f9e43694f40>>
Traceback (most recent call last):
  File "/home/scrapy/scrapy/lib/python3.8/site-packages/scrapy/utils/signal.py", line 43, in send_catch_log
    response = robustApply(
  File "/home/scrapy/scrapy/lib/python3.8/site-packages/pydispatch/robustapply.py", line 55, in robustApply
    return receiver(*arguments, **named)
  File "/home/scrapy/scrapy/lib/python3.8/site-packages/scrapyrt/core.py", line 181, in spider_idle
    spider.crawler.engine.crawl(self.request, spider)
TypeError: crawl() takes 2 positional arguments but 3 were given

```
I'm using scrapyrt with scrapy and scrapy-playwright.

**Comment by michaelbrunnbauer** [comment]
Never mind - I found the deprecation removals notice and guess scrapyrt  is just a piece of junk now.

**Comment by desprit** [comment]
@michaelbrunnbauer found what and where exactly? why not sharing?

**Comment by michaelbrunnbauer** [comment]
I think I was referring to the removal of the spider argument for crawl(). This was fixed with the  v0.14.0 release of scrapyrt on Aug 10, 2023, so I apologize for my wording and take it back.
